app/controllers/sign_up/recovery_codes_controller.rb

```diff
@@ -36,7 +36,7 @@ def next_step
       elsif current_user.password_reset_profile.present?
         reactivate_profile_path
       else
-        after_sign_in_path_for(current_user)
+        profile_path
```

Comment:
In isolation this change looks odd to me. Isn't the logic more subtle than simply always going to `profile_path`?